docs/evidence/general-mechanics/bugs.md

```diff
@@ -420,6 +420,24 @@ This can be also done without the negative edge N1, but requires a 1 frame M1 cl
 **Significance:**  
 The Archer class really is made up of archers!
 
+### Kuki Planting Things
+
+**By:** an odd medium\#6288  
+**Added:** <Version date="2022-12-27" />  
+**Last tested:** <VersionHl date="2022-12-27" />  
+[Discussion](https://tickets.deeznuts.moe/transcripts/kuki-planting-things)
+
+**Finding:**  
+Kuki Shinobu will overlay her equipped sword on top of her dagger if she casts her Elemental Burst in the middle of her N3.  
```

Comment:
Ticket author here: `middle of her N3` can be changed to `during her N3` as you have written in [docs/general-mechanics/bugs.md](https://github.com/KQM-git/TCL/pull/120/files#diff-5e755f556a9322e333ce8206daf45d4c1fcf317348ba14436ec58227779f527eR51). 
This was a good call on your part, and its much clearer as the timing doesn't necessarily have to be in the middle of N3.